README.md

```diff
@@ -1,265 +1,85 @@
-
-1. Install j2cli:
-
-    `$ pip install j2cli`
-
-1. Install CDI:
-
-    ```
-    $ cdi_namespace=default \
-      docker_prefix=kubevirt \
-      docker_tag=v1.1.1 \
-      pull_policy=IfNotPresent \
-      verbosity=1 \
-      j2 manifests/generated/cdi-controller.yaml.j2 | kubectl create -f -
-    ```
-    Check the template file and make sure to supply values for all variables.
-
-> Note: the default verbosity level is set to 1 in the controller deployment file, which is minimal logging. If greater details are desired increase the `-v` number to 2 or 3.
-
-> Note: the importer pod uses the same logging verbosity as the controller. If a different level of logging is required after the controller has been started, the deployment can be edited and applied by using `kubectl apply -f <CDI-MANIFEST>`. This will not alter the running controller's logging level but will affect importer pods created after the change. To change the running controller's log level requires it to be restarted after the deployment has been edited.
-
-
-### Start Importing Images
-
-> Note: The CDI controller is a required part of this work flow.
-
-Make copies of the [example manifests](./manifests/example) for editing. The necessary files are:
-- golden-pvc.yaml
-- endpoint-secret.yaml
-
-###### Edit golden-pvc.yaml:
-1.  `storageClassName:` The default StorageClass will be used if not set.  Otherwise, set to a desired StorageClass.
-
-1.  `cdi.kubevirt.io/storage.import.endpoint:` The full URL to the VM image in the format of: `http://www.myUrl.com/path/of/data` or `s3://bucketName/fileName`.
-
-1.  `cdi.kubevirt.io/storage.import.secretName:` (Optional) The name of the secret containing the authentication credentials required by the file server.
-
-###### Edit endpoint-secret.yaml (Optional):
-
-> Note: Only set these values if the file server requires authentication credentials.
-
-1. `metadata.name:` Arbitrary name of the secret. Must match the PVC's `cdi.kubevirt.io/storage.import.secretName:`
-
-1.  `accessKeyId:` Contains the endpoint's key and/or user name. This value **must be base64 encoded** with no extraneous linefeeds. Use `echo -n "xyzzy" | base64` or `printf "xyzzy" | base64` to avoid a trailing linefeed
-
-1.  `secretKey:` the endpoint's secret or password, again **base64 encoded** with no extraneous linefeeds.
-
-### Deploy the API Objects
+There are quite a few examples in the [example manifests](https://github.com/kubevirt/containerized-data-importer/tree/release-v1.5/manifests/example), check them out as a reference to create DataVolumes from additional sources like registries, S3 and your local system.
```

Comment:
I would remove the release part in the path. Something like 
```
[example manifests](./manifests/example)
```
 should do the trick.

Reply:
Agreed, updated.